core/src/main/resources/lib/layout/layout.jelly

```diff
@@ -227,52 +227,45 @@ ${h.initPageVariables(context)}
-        <div id="side-panel">
-          <div id="side-panel-content">
-            <j:set var="mode" value="side-panel" />
-            <d:invokeBody />
+      <div id="side-panel">
+        <j:set var="mode" value="side-panel" />
+        <d:invokeBody />
+  
+        <!-- add YUI logger if debugging YUI -->
+        <j:if test="${h.yuiSuffix=='debug'}">
+        <div id="yui-logreader" style="margin-top:1em"/>
+          <script>
+            Behaviour.addLoadEvent(function(){
+              var logReader = new YAHOO.widget.LogReader("yui-logreader");
+              logReader.collapse();
+            });
+          </script>
+        </j:if>
+      </div>
+      <div id="main-panel">
+         <j:set var="mode" value="main-panel" />
+         <d:invokeBody/>
+      </div>
+    </div>
 
-            <!-- add YUI logger if debugging YUI -->
-            <j:if test="${h.yuiSuffix=='debug'}">
-              <div id="yui-logreader" style="margin-top:1em"/>
-              <script>
-                Behaviour.addLoadEvent(function(){
-                  var logReader = new YAHOO.widget.LogReader("yui-logreader");
-                  logReader.collapse();
-                });
-              </script>
+    <footer>
+      <div class="container-fluid">
+        <div class="row">
+          <div class="col-md-6" id="footer"></div>
+          <div class="col-md-18">
+            <span class="page_generated">${%Page generated}: <i:formatDate value="${h.getCurrentTime()}" type="both" dateStyle="medium" timeStyle="medium"/></span>
+            <span class="rest_api"><a href="api/">REST API</a></span>
+            <span class="jenkins_ver"><a href="${h.getFooterURL()}">Jenkins ver. ${h.version}</a></span>
+            <j:if test="${extensionsAvailable}">
+            <j:forEach var="pd" items="${h.pageDecorators}">
+              <st:include it="${pd}" page="footer.jelly" optional="true" />
+            </j:forEach>
             </j:if>
           </div>
         </div>
-
-        <div id="main-panel">
-          <div id="main-panel-content">
-            <j:set var="mode" value="main-panel" />
-            <d:invokeBody/>
-          </div>
-        </div>
-      </div>
-    </div>
-    <div id="footer-container" class="hidden-xs">
-      <div id="footer">
-        <span class="page_generated">
-          ${%Page generated}:
-          <i:formatDate value="${h.getCurrentTime()}" type="both" dateStyle="medium" timeStyle="medium"/>
-        </span>
-        <span class="rest_api">
-          <a href="api/">REST API</a>
-        </span>
-        <span class="jenkins_ver">
-          <a href="${h.getFooterURL()}">Jenkins ver. ${h.version}</a>
-        </span>
-        <j:if test="${extensionsAvailable}">
-        <j:forEach var="pd" items="${h.pageDecorators}">
-          <st:include it="${pd}" page="footer.jelly" optional="true" />
-        </j:forEach>
-        </j:if>
       </div>
-    </div>
+    </footer>
+
   </body>
 </html>
 </j:jelly>
```

Comment:
Sure, It is not easy to see the changes. Basically, I've tried to simplify the markup.